test/e2e/kubernetes/kubernetes_test.go

```diff
@@ -823,10 +823,10 @@ var _ = Describe("Azure Container Cluster using the Kubernetes Orchestrator", fu
 
 		It("should be able to attach azure file", func() {
 			if eng.HasWindowsAgents() {
-				if common.IsKubernetesVersionGe(eng.ClusterDefinition.ContainerService.Properties.OrchestratorProfile.OrchestratorVersion, "1.11") {
-					// Failure in 1.11+ - https://github.com/kubernetes/kubernetes/issues/65845
-					Skip("Kubernetes 1.11 has a known issue creating Azure PersistentVolumeClaims")
-				} else if common.IsKubernetesVersionGe(eng.ClusterDefinition.ContainerService.Properties.OrchestratorProfile.OrchestratorVersion, "1.8") {
+				if eng.ExpandedDefinition.Properties.OrchestratorProfile.OrchestratorVersion == "1.11.0" {
+					// Failure in 1.11.0 - https://github.com/kubernetes/kubernetes/issues/65845, fixed in 1.11.1
+					Skip("Kubernetes 1.11.0 has a known issue creating Azure PersistentVolumeClaims")
+				} else if common.IsKubernetesVersionGe(eng.ExpandedDefinition.Properties.OrchestratorProfile.OrchestratorVersion, "1.8") {
```

Comment:
I'm not sure "1.8" is a valid version, I think you need "1.8.0"

Reply:
Ah yes you're right.